companies/vivid-money.json

```diff
@@ -0,0 +1,18 @@
+{
+    "slug": "vivid-money",
+    "relevant-countries": [
+        "de"
```

Comment:
According to the country selector in the footer, they also target a few other countries. We should add those as well.

Reply:
sure, thanks for your feedback!